tools/cli/bin/jetpack

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env node
+
+/**
+ * Standardizes the cwd for the process. Allows `jetpack` cli to run correctly from any location in the repo.
+ */
+const process = require( 'process' );
+const path = require( 'path' );
+
+process.chdir( path.join( __dirname, '../../..' ) );
+
+try {
+	// eslint-disable-next-line no-global-assign
+	require = require( 'esm' )( module /*, options*/ );
+	require( '../cliRouter' ).cli();
```

Comment:
The `require( '../cliRouter' )` has a noticeable delay (~0.8 seconds), which seems odd to me.

Reply:
Do you have a particular favorite tool to time it? My execution times are at about 0.3 seconds total.

Reply:
I just did something along the lines of
```
let t0 = Date.now();
...
console.log( Date.now() - t0 );
```